ReactApp/src/contexts/user-context.js

```diff
@@ -52,6 +54,7 @@ export const UserProvider = ({ children }) => {
     const fetchUsers = async () => {
       try {
         const response = await requestServer(serverPath.GET_ALL_USERS, httpsMethod.GET);
+        response.users = response.users.filter(user => user.type !== USERS.ADMIN)
```

Comment:
Why filter out admins?

Reply:
To avoid admin editing himself

Reply:
what's the problem with admin editing itself? If anything it shouldn't edit any admin BUT himself!
Also an admin should be able to track other admin users in the system

Reply:
Isn't admin a unique user in the system? that's how I imagined it would be. If we have multiple admins in the system, then I agree.

Reply:
May be unique, still he should be able to edit any aspect of himself. Only things he can't do is change his user-type or delete himself.